data/json/items/comestibles/egg.json

```
@@ -352,6 +352,18 @@
     "rot_spawn": "GROUP_EGG_LADY_BUG",
     "rot_spawn_chance": 50
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_lady_bug_giant",
+    "name": { "str": "giant lady bug egg" },
```

```suggestion
    "name": { "str": "giant ladybug egg" },
```

This is standard spelling in the US https://en.wiktionary.org/wiki/ladybug